Snowflake.Data/Core/FileTransfer/SFFileTransferAgent.cs

```diff
@@ -785,6 +794,12 @@ private static IEnumerable<string> ExpandDirectories(string directoryPath)
             {
                 if (ContainsWildcard(part))
                 {
+                    // Directory containing the wildcard is the first one in the path
+                    if (resolvedPaths.Count == 0)
+                    {
+                        resolvedPaths.Add("./");
```

Comment:
If we have directoryPath like A*/B*/... and there is no directory A*, resolvedPaths will be cleared in 811 line. So we will enter here also for second part B* and I think it's not ok.

Reply:
Question: Does "./" work on Windows?

Reply:
> If we have directoryPath like A*/B*/... and there is no directory A*, resolvedPaths will be cleared in 811 line. So we will enter here also for second part B* and I think it's not ok.

I see, I'll make changes and try to make it work with that case

> Question: Does "./" work on Windows?

It does, would it be safer to use something like `Directory.GetCurrentDirectory()` so it can work with all OS?

Reply:
I've updated the PR and made a test for the A*/B* case